tests/aws/services/events/test_events_rules.py

```diff
@@ -1,345 +0,0 @@
-"""Tests for EventBridge rules.
-Tests for rule routing of events as well as rule creation and deletion.
-"""
-
-import json
-
-import pytest
-
-from localstack.testing.config import TEST_AWS_ACCOUNT_ID, TEST_AWS_REGION_NAME
-from localstack.testing.pytest import markers
-from localstack.utils.aws import arns
-from localstack.utils.strings import short_uid
-from localstack.utils.sync import poll_condition
-from tests.aws.services.events.helper_functions import (
-    assert_valid_event,
-    is_v2_provider,
-    sqs_collect_messages,
-)
-from tests.aws.services.events.test_events import TEST_EVENT_PATTERN
-
-
-@markers.aws.validated
-def test_put_rule(aws_client, snapshot, clean_up):
-    rule_name = f"rule-{short_uid()}"
-    snapshot.add_transformer(snapshot.transform.regex(rule_name, "<rule-name>"))
-
-    response = aws_client.events.put_rule(
-        Name=rule_name, EventPattern=json.dumps(TEST_EVENT_PATTERN)
-    )
-    snapshot.match("put-rule", response)
-
-    response = aws_client.events.list_rules(NamePrefix=rule_name)
-    snapshot.match("list-rules", response)
-    rules = response["Rules"]
-    assert len(rules) == 1
-    assert json.loads(rules[0]["EventPattern"]) == TEST_EVENT_PATTERN
-
-    clean_up(rule_name=rule_name)
-
-
-@markers.aws.validated
-def test_rule_disable(aws_client, clean_up):
```

Comment:
better coverage in `aws.services.events.test_events.TestEventRule.test_disable_re_enable_rule` 💯 
🧹🧹🧹